docs/transport.md

```
@@ -0,0 +1,251 @@
+// packages/hd-transport/src/constants.ts
+export const MESSAGE_TOP_CHAR = 0x003f;        // '?' chunk marker
+export const MESSAGE_HEADER_BYTE = 0x23;       // '#' protocol header
+export const HEADER_SIZE = 8;                  // Protocol header size
+export const BUFFER_SIZE = 63;                 // Data per chunk
+```
+
+## Message Protocol
+
+### Protocol Frame Structure
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)